### PR TITLE
Minor fixes and refinements

### PR DESCRIPTION
**Pull request by arthurattwell**
This includes a bunch of small fixes and refinements, mostly to CSS, drawn from real-life projects we've worked on over the last few months.

- Add src-set to figure includes
- Wrap pagination markers in spans for more CSS control
- Move button and sidenote mixins to dedicated partials
- Load new partials with utility mixins
  so that they can be included in other features more easily.
- Create pullquote mixin for easier customisation
- Remove deprecated print figure classes and add missing ones
- Remove opinionated vertical writing mode for side margin boxes
- Make sidebar variables global and add sidebar() mixin
  for moving any custom element into sidebar easily.
- Remove variables from book sass to always start with project defaults
  In practice we've found that this is best practice,
  since for most projects/repos we keep styles consistent across books.
- Improve accordion heading sizes, removing unnecessary media query
- Create variable for adjusting paragraph indent globally
- Don't style for all inputs, only type=text
- Improve spacing of nav-buttons to align with other features
- Fix video play-button cursor to pointer
- Fix errors in side margin box variable names
- Fix samples page numbering
- Add missing variable values
- Refine default styles for question feedback boxes
- Set default margins to align with baseline grid

### CODE REVIEW

**Comment by arthurattwell** [comment]
@LouiseSteward Can you glance an eye over these changes looking for obvious or silly mistakes? I think I've tested everything here properly.

File: _sass/partials/_epub-base-typography.scss [hunk 1]
```diff
@@ -57,7 +57,7 @@ $epub-base-typography: true !default;
     p {
         margin: 0;
         & + p {
-            text-indent: $line-height-default;
+            text-indent: $paragraph-indent;
```
Comment (LouiseSteward) [inline _sass/partials/_epub-base-typography.scss]: We're changing the default paragraph indent from `line-height-default` to `font-size-default` - just want to check that this was intended.

File: _sass/print-pdf.scss [hunk 2]
```diff
@@ -355,6 +355,9 @@ $print-page-setup-lightning-source: false !default; // Removes crop marks and se
 // Reset PrinceXML CSS
 @import "partials/print-css-reset";
 
+// Load general-purpose mixins
```
[the remark this answers is not in the document]
Comment (LouiseSteward) [reply _sass/print-pdf.scss]: Never mind - you caught this in a later commit :)

File: _sass/partials/_epub-base-typography.scss (hunk 1 above)
Comment (arthurattwell) [reply _sass/partials/_epub-base-typography.scss]: Thanks, yes, this was deliberate. Over time and drawing from Karen's designs, I've noticed that for most projects the line height is a bit too much, and proportionally the font size makes for a more elegant indent.

File: _sass/partials/_print-pullquotes.scss [hunk 3]
```diff
@@ -1,17 +1,21 @@
+@mixin pullquote($font-size: $font-size-default * 1.4, $line-height: $line-height-default) {
```
[the remark this answers is not in the document]
Comment (arthurattwell) [reply _sass/partials/_print-pullquotes.scss]: Thanks, yes. The PDF font and line height variables are set in absolute units (usually points) while the others use proportional units (usually ems, though rems may be better). Hence the difference here.

**Comment by arthurattwell** [comment]
Thanks, @louisesteward, I really appreciate the close eye there.

**Pull request merged**